### PR TITLE
LPS-130102 Wiki insert link is showing the wrong link dialog

### PR DESCRIPTION
**Pull request by boton**
Hi @liferay-frontend,

**Description:**

When trying to add a new link in a wiki (creole) the generic link dialog is shown instead of the [wikilinks](https://github.com/liferay/liferay-portal/blob/c63c92cc3904d262e385dbdff50b35b21c18272a/modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/wikilink/dialogs/link.js).

**How to test it**

1. Go to: Product Menu > Content & Data > Wiki > Main > [+] (New Wiki Page)
2. Click on the link icon
 

**Before the fix**
Show the generic link dialog, throw browser error.

image

**After the fix**
Show the specific wiki link dialog.

image

### CODE REVIEW

**Comment by ambrinchaudhary** [comment]
Hi @boton, as we saw before, I'm ok with the change. But not sure why are you sending this over to FI... as this is specific of wiki and therefore of Lima...

**Review by ambrinchaudhary — APPROVED** [review]

**Comment by ambrinchaudhary** [comment]
ci:forward

**Comment by boton** [comment]
> Hi @boton, as we saw before, I'm ok with the change. But not sure why are you sending this over to FI... as this is specific of wiki and therefore of Lima...

Sorry @ambrinchaudhary, it's just because the files belong to FI I didn't really think about it 🤔

File: modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/java/com/liferay/frontend/editor/ckeditor/web/internal/editor/configuration/CKEditorCreoleConfigContributor.java [hunk 1]
```diff
@@ -85,7 +85,7 @@ public void populateConfigJSONObject(
 		StringBundler sb = new StringBundler(4);
 
 		sb.append("bidi,codemirror,colorbutton,colordialog,div,");
-		sb.append("elementspath,font,forms,indentblock,justify,keystrokes");
+		sb.append("elementspath,font,forms,indentblock,justify,keystrokes,");
```
Comment (jbalsas) [inline modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/java/com/liferay/frontend/editor/ckeditor/web/internal/editor/configuration/CKEditorCreoleConfigContributor.java:88]: nice catch! 😱

Comment (boton) [reply modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/java/com/liferay/frontend/editor/ckeditor/web/internal/editor/configuration/CKEditorCreoleConfigContributor.java:88]: With the help of @ambrinchaudhary, it has been very easy ❤️

**Pull request closed without merging**